Dockerfile

```diff
@@ -23,7 +23,7 @@
 # Dockerfile for building HMS SLS.
 
 # Build base just has the packages installed we need.
-FROM arti.dev.cray.com/baseos-docker-master-local/golang:1.16-alpine3.13 AS build-base
+FROM artifactory.algol60.net/csm-docker/stable/docker.io/library/golang:1.16-alpine3.13 AS build-base
```

Comment:
can you change the last part of this? the 3.13 needs to be updated.   I think FAS has an example for the golang bit

Reply:
Updated